src/pymor/algorithms/pod.py

```diff
@@ -68,43 +71,16 @@ def pod(A, modes=None, product=None, rtol=4e-8, atol=0., l2_err=0.,
     assert len(A) > 0
     assert modes is None or modes <= len(A)
     assert product is None or isinstance(product, OperatorInterface)
+    assert method in ('mos', 'qr_svd')
 
     logger = getLogger('pymor.algorithms.pod.pod')
 
-    with logger.block(f'Computing Gramian ({len(A)} vectors) ...'):
-        B = A.gramian(product)
-
-        if symmetrize:     # according to rbmatlab this is necessary due to rounding
-            B = B + B.T
-            B *= 0.5
-
-    with logger.block('Computing eigenvalue decomposition ...'):
-        eigvals = None if (modes is None or l2_err > 0.) else (len(B) - modes, len(B) - 1)
-
-        EVALS, EVECS = eigh(B, overwrite_a=True, turbo=True, eigvals=eigvals)
-        EVALS = EVALS[::-1]
-        EVECS = EVECS.T[::-1, :]  # is this a view? yes it is!
-
-        tol = max(rtol ** 2 * EVALS[0], atol ** 2)
-        above_tol = np.where(EVALS >= tol)[0]
-        if len(above_tol) == 0:
-            return A.space.empty(), np.array([])
-        last_above_tol = above_tol[-1]
-
-        errs = np.concatenate((np.cumsum(EVALS[::-1])[::-1], [0.]))
-        below_err = np.where(errs <= l2_err**2)[0]
-        first_below_err = below_err[0]
-
-        selected_modes = min(first_below_err, last_above_tol + 1)
-        if modes is not None:
-            selected_modes = min(selected_modes, modes)
-
-        SVALS = np.sqrt(EVALS[:selected_modes])
-        EVECS = EVECS[:selected_modes]
-
-    with logger.block(f'Computing left-singular vectors ({len(EVECS)} vectors) ...'):
-        POD = A.lincomb(EVECS / SVALS[:, np.newaxis])
+    svd_va = mos if method == 'mos' else qr_svd
+    POD, SVALS, _ = svd_va(A, product=product, modes=modes, rtol=rtol, atol=atol,
+                           l2_err=l2_err, check=False)
 
+    if orthonormalize is None:
+        orthonormalize = True if method == 'mos' else False
```

Comment:
So we only want to re-orthonormalize when we use `mos` as a backend. At the same time, we re-orthonormalize in `mos` by default, leading to a double re-orthonormalization. I think, `orthonormalize` should be removed from the signature of `pod`. The same seems true for `check`.

I think we should make `pod` just a thin wrapper around the SVD algorithms for people insisting to look for a pod method (with the added benefit that the SVD algorithm can be exchanged without changing the code). We could consider adding an `svd_params` dict argument which is passed to the SVD implementation via `**svd_params` similar to what is done in `rb_greedy`.